Sources/MongoSwift/ChangeStream.swift

```diff
@@ -195,6 +193,26 @@ public class ChangeStream<T: Codable>: CursorProtocol {
         }
     }
 
+    /**
+     * Calls the provided closure with each event in the change stream as it arrives.
+     *
+     * - Returns:
+     *     An `EventLoopFuture<Void>` which will succeed once the change stream is killed via `kill`.
```

Comment:
this feels like a weird way to put this as at first read it seems unrelated to what this method does, but I'm not sure how else to say this, any ideas?

Reply:
The change stream can be closed via "invalidate" events too ([docs](https://docs.mongodb.com/manual/changeStreams/#open-a-change-stream)). Might be better to just say "... which will resolve/complete once the change stream is closed."